integrationtest/automatictest/auto-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Licensed under the Apache License, Version 2.0
+# See accompanying LICENSE file.
+
+# Root directory, put the Gearpump source code in this directory.
+# The report directory and report.zip will be generated in this directory, after integration test.
+rootDir="/root/integrationtest"
+
+# The gearpump repo
+gearpumpRepo="https://github.com/gearpump/gearpump.git"
+
+# The mail recipients list. If more than one, split by space.
+sendTo="kewei.k.sun@intel.com"
```

Comment:
Please add another file called maillist, and add all mails into the mail list.
one line one mail.